hw/model/src/fpga_regs.rs

```diff
@@ -12,11 +12,16 @@ register_bitfields! {
         CptraSsRstB OFFSET(1) NUMBITS(1) [],
         CptraObfUdsSeedVld OFFSET(2) NUMBITS(1) [],
         CptraObfFieldEntropyVld OFFSET(3) NUMBITS(1) [],
-        Rsvd0 OFFSET(4) NUMBITS(1) [],
-        Rsvd1 OFFSET(5) NUMBITS(1) [],
-        BootfsmBrkpoint OFFSET(6) NUMBITS(1) [],
-        SsDebugIntent OFFSET(7) NUMBITS(1) [],
-        I3cAxiUserIdFiltering OFFSET(8) NUMBITS(1) [],
+        RsvdDbgLocked OFFSET(4) NUMBITS(1) [],
```

Comment:
I don't think we actually use these anymore (we use the ones in caliptra-sw/hw-model/)